cip/CIP2015-06-24-call-procedures.adoc

```diff
@@ -0,0 +1,250 @@
+= CIP2015-06-24 Calling Stored Procedures
+:numbered:
+:toc:
+:toclevels: 3
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Stefan Plantikow <stefan.plantikow@neotechnology.com>
+*Author:* Nigel Small <nigel.small@neotechnology.com>
+*Author:* Petra Selmer <petra.selmer@neotechnology.com>
```

Comment:
I think my name can be removed from the author list, as I had very little to do with this CIP :-)